### PR TITLE
add tooltips

### PR DESCRIPTION
**Pull request by mshakira**
- add tooltips
- summaries make it all left aligned
- style the fonts for tooltip

Screenshot 2023-10-31 at 11 47 22 AM
Screenshot 2023-10-31 at 11 47 14 AM
Screenshot 2023-10-31 at 11 47 08 AM
Screenshot 2023-10-31 at 11 47 03 AM
Screenshot 2023-10-31 at 11 48 47 AM
Screenshot 2023-10-31 at 11 48 38 AM
Screenshot 2023-10-31 at 11 48 29 AM

### CODE REVIEW

File: ui/src/components/common/SummaryPageLayout/partials/SummaryStatuses/index.tsx [hunk 1]
```diff
@@ -40,6 +43,22 @@ export function SummaryStatuses({
...
+      {tooltip && (
+        <Box
+          sx={{
+            display: "flex",
+            flexDirection: "row",
+            alignItems: "right",
+            width: "100%",
+            justifyContent: "flex-end",
+            paddingRight: "0.2rem",
+            marginBottom: "-1rem",
```
Comment (darshansimha) [inline ui/src/components/common/SummaryPageLayout/partials/SummaryStatuses/index.tsx:55]: Is there anyway we can avoid negative margins?

File: ui/src/components/common/SummaryPageLayout/partials/SummaryTitledValue/index.tsx [hunk 2]
```diff
@@ -19,6 +25,22 @@ export function SummaryTitledValue({ title, value }: SummaryTitledValueProps) {
...
+      {tooltip && (
+        <Box
+          sx={{
+            display: "flex",
+            flexDirection: "row",
+            alignItems: "right",
+            width: "100%",
+            justifyContent: "flex-end",
+            paddingRight: "0.2rem",
+            marginBottom: "-1rem",
```
Comment (darshansimha) [inline ui/src/components/common/SummaryPageLayout/partials/SummaryTitledValue/index.tsx:37]: +1

File: ui/src/components/common/SummaryPageLayout/partials/SummaryStatuses/index.tsx (hunk 1 above)
Comment (mshakira) [reply ui/src/components/common/SummaryPageLayout/partials/SummaryStatuses/index.tsx]: removed -ve margin and refactored it

File: ui/src/components/common/SummaryPageLayout/partials/SummaryTitledValue/index.tsx (hunk 2 above)
Comment (mshakira) [reply ui/src/components/common/SummaryPageLayout/partials/SummaryTitledValue/index.tsx]: removed -ve margin and refactored it

**Review by darshansimha — APPROVED** [review]

**Pull request merged**